react/features/chat/components/AbstractChat.js

```diff
@@ -106,6 +113,9 @@ export function _mapStateToProps(state: Object) {
     const _localParticipant = getLocalParticipant(state);
 
     return {
+        _isModal: isMobileBrowser()
+            ? window.innerWidth <= MOBILE_SMALL_WIDTH_THRESHOLD
```

Comment:
Does this work when resizing dynamically? I wonder if these props will be computed at the right time.

Reply:
Yup, dragging past the thresholds switches between the sidebar view and the modal one